ngraph/core/include/openvino/core/function.hpp

```diff
@@ -91,7 +91,20 @@ class OPENVINO_API Function {
     /// Return the op that generates output i
     std::shared_ptr<ngraph::Node> get_output_op(size_t i) const;
 
-    ngraph::Output<ngraph::Node> output(size_t i) const;
+    /// Output functions
+    ov::Output<ov::Node> output(size_t i) const;
```

Comment:
```suggestion
    ov::Output<ov::Node> output(const size_t i) const;
```

Reply:
My suggestion is to keep 'size_t', not 'const size_t' here. When passing arguments by value, this 'const' in function declaration is meaningless, thus sometimes not recommended to use. https://abseil.io/tips/109
```
void F(const int x);          // 1: declares F(int)
void F(int* const x);         // 2: declares F(int*)
void F(const int* const x);   // 3: declares F(const int*)
```
However in '.cpp' code, just to protect yourself (to ensure that you're not changing `i` inside function definition) sometimes 'const' is used for arguments passed by value. Personally I prefer to use 'size_t' in ".cpp" files as well as it looks a bit confusing having `size_t` in .hpp and `const size_t in .cpp

Reply:
Makes sense, I wasn't aware of this behavior described in the article, thanks.